src/http.cc

```diff
@@ -82,9 +82,7 @@ HttpStateData::HttpStateData(FwdState *theFwdState) :
 {
     debugs(11,5, "HttpStateData " << this << " created");
     serverConnection = fwd->serverConnection();
```

Comment:
We can move this initialization to the initialization list above (where it belongs), but let's not do that in this PR unless explicitly requested.